src/main/java/com/stripe/model/Charge.java

```diff
@@ -602,6 +606,473 @@ public void setRuleObject(Rule rule) {
+    CardPresent cardPresent;
+
+    @SerializedName("eps")
+    Eps eps;
+
+    @SerializedName("giropay")
+    Giropay giropay;
+
+    @SerializedName("ideal")
+    Ideal ideal;
+
+    @SerializedName("multibanco")
+    Multibanco multibanco;
+
+    @SerializedName("p24")
+    P24 p24;
+
+    @SerializedName("sepa_debit")
+    SepaDebit sepaDebit;
+
+    @SerializedName("sofort")
+    Sofort sofort;
+
+    @SerializedName("type")
+    String type;
+
+    @SerializedName("wechat")
+    Wechat wechat;
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class Account extends StripeObject {}
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class AchCreditTransfer extends StripeObject {
+      @SerializedName("account_number")
+      String accountNumber;
+
+      @SerializedName("bank_name")
+      String bankName;
+
+      @SerializedName("routing_number")
+      String routingNumber;
+
+      @SerializedName("swift_code")
+      String swiftCode;
+    }
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class AchDebit extends StripeObject {
+      @SerializedName("account_holder_type")
+      String accountNumber;
+
+      @SerializedName("bank_name")
+      String bankName;
+
+      @SerializedName("country")
+      String country;
+
+      @SerializedName("fingerprint")
+      String fingerprint;
+
+      @SerializedName("last4")
+      String last4;
+
+      @SerializedName("routing_number")
+      String routingNumber;
+    }
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class Alipay extends StripeObject {}
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class Bancontact extends StripeObject {
+      @SerializedName("bank_code")
+      String bankCode;
+
+      @SerializedName("bank_name")
+      String bankName;
+
+      @SerializedName("bic")
+      String bic;
+
+      @SerializedName("iban_last4")
+      String ibanLast4;
+
+      @SerializedName("preferred_language")
+      String preferredLanguage;
+
+      @SerializedName("verified_name")
+      String verifiedName;
+    }
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class Bitcoin extends StripeObject {
+      @SerializedName("bank_code")
+      String bankCode;
+
+      @SerializedName("bank_name")
+      String bankName;
+
+      @SerializedName("bic")
+      String bic;
+
+      @SerializedName("iban_last4")
+      String ibanLast4;
+
+      @SerializedName("preferred_language")
+      String preferredLanguage;
+
+      @SerializedName("verified_name")
+      String verifiedName;
+    }
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class Card extends StripeObject {
+      @SerializedName("brand")
+      String brand;
+
+      @SerializedName("checks")
+      PaymentMethod.Card.Checks checks;
```

Comment:
In payflow code, there are actually two classes of checks. One in the details here, and the other one in payment method, eventhough the schemas are exactly the same
```
lib/payment_flows/private/payment_methods/card/checks/details_api_resource.rb
lib/payment_flows/private/payment_methods/card/checks/api_resource.rb 	
```